src/arch/x86/cpuid.hh

```diff
@@ -38,28 +41,41 @@ class ThreadContext;
 
 namespace X86ISA
 {
-    struct CpuidResult
-    {
-        uint64_t rax;
-        uint64_t rbx;
-        uint64_t rcx;
-        uint64_t rdx;
 
-        // These are not in alphebetical order on purpose. The order reflects
-        // how the CPUID orders the registers when it returns results.
-        CpuidResult(uint64_t _rax, uint64_t _rbx,
-                    uint64_t _rdx, uint64_t _rcx) :
-            rax(_rax), rbx(_rbx), rcx(_rcx), rdx(_rdx)
-        {}
+struct CpuidResult
```

Comment:
seems you removed indentation here?

Reply:
Yes that is true -  I made an "undintation" to this PR to be more in line with the namespace changes that occurred in gem5 v22 (around that time).